notebooks/00020_fix_positions.ipynb

```diff
@@ -0,0 +1,2090 @@
+{
```

Comment:
<p>@dianagroch can you please check this? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdenkas/human-tafazzin-variants-database/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>